templates/inventory_cluster_4.1.j2

```diff
@@ -12,7 +12,7 @@
 {% if groups['execution']|default([])|length > 0 %}
 [execution_nodes]
 {% for execution_node in groups['execution'] %}
-{{ execution_node }}
+{{ execution_node }} node_type={{ execution_plane_node_type | default('execution') }}
```

Comment:
With this all the nodes in execution_nodes group would either be execution or hop node?
But with a hop node we need atleast 1 execution node in the end right ?

Reply:
yeah totally correct. let me fix it

Reply:
a new commit https://github.com/redhat-performance/ibm_cloud_tower_deploy/pull/18/commits/34094f1ea69ee8a4101208652f895bf5f36a8b3b has been pushed to fix the either or situation. Now I have added flexibility to support different `node_type` to different instances in the inventory